### PR TITLE
some contributions

### PR DESCRIPTION
**Pull request by cfoellmann**
fix (non-)static issues with domain_mapping_siteurl() and get_original_url()
- more

### CODE REVIEW

File: domain_mapping.php [hunk 1]
```diff
@@ -191,10 +191,10 @@ public function domain_mapping_siteurl( $setting ) {
 			$s = $wpdb->suppress_errors();
 
 			if ( get_site_option( 'dm_no_primary_domain' ) == 1 ) {
-				$domain = $wpdb->get_var( "SELECT domain FROM {$wpdb->dmtable} WHERE blog_id = '{$wpdb->blogid}' AND domain = '" . $wpdb->escape( $_SERVER[ 'HTTP_HOST' ] ) . "' LIMIT 1" );
+				$domain = $wpdb->get_var( $wpdb->prepare( "SELECT domain FROM {$wpdb->dmtable} WHERE blog_id = '{$wpdb->blogid}' AND domain = '" . $_SERVER[ 'HTTP_HOST' ] . "' LIMIT 1" ) );
```
Comment (markoheijnen) [inline domain_mapping.php]: Prepare is incorrectly used here.

File: inc/admin.php [hunk 2]
```diff
@@ -157,25 +157,29 @@ public function manage_page() {
...
+
 		global $wpdb, $parent_file;
 
 		$url = add_query_arg( array( 'page' => 'domainmapping' ), admin_url( $parent_file ) );
 
 		if ( ! empty( $_POST[ 'action' ] ) ) {
-			$domain = $wpdb->escape( $_POST[ 'domain' ] );
 
-			if ( $domain == '' )
+			$domain = $_POST[ 'domain' ];
```
Comment (markoheijnen) [inline inc/admin.php]: I believe sanitize_text_field() is need to be used here.

File: domain_mapping.php (hunk 1 above)
Comment (cfoellmann) [inline domain_mapping.php]: fixed in https://github.com/cfoellmann/wordpress-mu-domain-mapping/commit/f0cc4f09b12dfbd7d6133d8e700d27208aa6b0a5

File: inc/admin.php (hunk 2 above)
Comment (cfoellmann) [inline inc/admin.php]: added in https://github.com/cfoellmann/wordpress-mu-domain-mapping/commit/9c237dbf5ccc3371c6d3809abf320c0d681b5bfb

**Pull request closed without merging**